src/doc/rustdoc/src/what-is-rustdoc.md

```diff
@@ -93,6 +93,22 @@ passes `-L`, a flag that helps rustdoc find the dependencies
 your code relies on. If our project used dependencies, we'd get
 documentation for them as well!
 
+## Crate level documentation
+From the above examples, you can see that using the `///` syntax
+documents the next item. But that can't be used when you are
+documenting the entire crate, because there is nothing before the crate.
+It is where everything is placed under. So, in order to document an entire
+crate, you use this `//!` syntax. For example
```

Comment:
```suggestion
documenting the entire crate, because nothing comes before it: it is the
root of the crate. So in order to document an entire crate, you need to use `//!` syntax. For example:
```

Reply:
You marked this as resolved but didn't apply the suggestion.

Reply:
You did it again 😆

Reply:
You still have not resolved this comment. If you don't think this is a good change to make, please explain why instead of continuing to ignore it.

Reply:
You still have not addressed this comment. If you don't think this is a good change to make, please explain why instead of ignoring it.